docs/fundamentals/aws-sdk-rust.md

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_position: 1
+title: AWS SDK for Rust
+description: The AWS SDK for Rust
+keywords: [fundamentals]
+---
+
+## Introduction
+
+The [AWS SDK for Rust](https://aws.amazon.com/sdk-for-rust/) has been developed to simplify operations with various AWS services including DynamoDB, S3, SQS and many others.  More importantly, it comes with sensible and battle-tested defaults that builders can take advantage of such as built-in retries. It has also been developed to minimize resource allocation with wasted copies and clones giving you more resources for your applications.  And finally, it's feature aware so you can include just the crates you need to minimize build time and final binary output size.
```

Comment:
Maybe add a hyperlink for the DynamoDB word to the page on the SDK?

Also, what do you think about replacing the word 'builders' with 'you'? Talk directly to the reader?